<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:rgb(40, 44, 50);
color:white;
}

QFrame{
border:sold 10px rgba(0 ,0, 0)
}


QLineEdit{
background-color:rgb(58. 104. 148);
}

QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}


QLineEdit{
border: 1px solid #A0A0A0;
background-color:rgb(65, 65, 65);
color:white;
}


QLineEdit:hover { 
	border: 1px solid #298DFF;
	border-radius: 3px;
	selection-color: #F2F2F2;
}

QLineEdit[EchoMode=&quot;2&quot;] {lineedit-password-character:42;}
</string>
  </property>
  <widget class="QToolButton" name="minimizeToolButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>-10</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
  <widget class="QToolButton" name="quitToolButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="PicLineEdit" name="phonePicLineEdit" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>175</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="PicLineEdit" name="pswPicLineEdit" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>325</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="PicLineEdit" name="repswPicLineEdit" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="PicLineEdit" name="emailPicLineEdit" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="PicLineEdit" name="namePicLineEdit" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="registerPushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>minimizeToolButton</zorder>
  <zorder>quitToolButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PicLineEdit</class>
   <extends>QWidget</extends>
   <header>piclineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
